klaviyo/api_helper.py

```diff
@@ -125,6 +125,18 @@ def _build_query_string(self, params, is_test):
             'test': 1 if is_test else 0,
         })
 
+    def _build_data_string(self, params):
+        '''Format params into a data string to pass into data field of url-encoded form
+
+        Args:
+            params (dict): params to convert.
+
+        Returns:
+            (str): data string to pass into data field of url-encoded form.
+        '''
```

Comment:
params to convert? what params are these? utm?
```suggestion
        """Format params into a data string to pass into data field of url-encoded form.

        Args:
            params (dict): Params to convert.

        Returns:
            (str): Data string to pass into data field of url-encoded form.
        """
```